ddtrace/internal/symbol_db/symbols.py

```diff
@@ -531,10 +531,32 @@ class SymbolDatabaseUploader(BaseModuleWatchdog):
     def __init__(self) -> None:
         super().__init__()
 
+        self._seen_modules: t.Set[str] = set()
+        self._update_called = False
+
+        self._process_unseen_loaded_modules()
+
+    def _process_unseen_loaded_modules(self) -> None:
         # Look for all the modules that are already imported when this is
         # installed and upload the symbols that are marked for inclusion.
         context = ScopeContext()
-        for module in (_ for _ in list(sys.modules.values()) if is_module_included(_)):
+        for name, module in list(sys.modules.items()):
+            # Skip modules that are being initialized as they might not be
+            # fully loaded yet.
+            try:
+                if module.__spec__._initializing:  # type: ignore[union-attr]
+                    continue
```

Comment:
Should we retry them?

Reply:
That's what the extra update will do 🙂

Reply:
could modules get loaded after RC config upgrade was received? an application load more modules dynamically over time?

Reply:
For those we already have the import hook. Here we want to make sure that we catch those that might have been imported while we are installing the uploader